diff -urN gtk+licq-0.39.orig/src/options_dialog.cpp gtk+licq-0.39/src/options_dialog.cpp
--- gtk+licq-0.39.orig/src/options_dialog.cpp	Sat Jul  1 17:55:29 2000
+++ gtk+licq-0.39/src/options_dialog.cpp	Wed Sep 27 10:22:10 2000
@@ -4,6 +4,7 @@
 #include <stdio.h>
 #include <sys/types.h>
 #include <dirent.h>
+#include <unistd.h>
 
 #include "licq_sar.h"
 
diff -urN gtk+licq-0.39.orig/src/plugin_dialog.cpp gtk+licq-0.39/src/plugin_dialog.cpp
--- gtk+licq-0.39.orig/src/plugin_dialog.cpp	Tue Mar 28 01:59:06 2000
+++ gtk+licq-0.39/src/plugin_dialog.cpp	Wed Sep 27 10:21:55 2000
@@ -5,6 +5,7 @@
 #include <sys/types.h>
 #include <dirent.h>
 #include <dlfcn.h>
+#include <unistd.h>
 
 #include "interface.h"
 #include "contactlist.h"
